db.go

```diff
@@ -90,27 +91,35 @@ func (db *DB) ActivateLoginToken(ctx context.Context, token string) (string, err
 
 // CreateSession creates a new session for the user ID and returns the
 // session key
-func (db *DB) CreateSession(ctx context.Context, userID, userAgent, ip string) (string, error) {
+func (db *DB) CreateSession(ctx context.Context, userID, userAgent, ip string) (email string, key string, err error) {
```

Comment:
hmm @bradleyfalzon - is there currently a way to remove apicompat from these checks?